x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.tsx

```diff
@@ -125,10 +132,12 @@ export const createRegisteredAttachmentUserActionBuilder = <
         actions: (
           <UserActionContentToolbar id={comment.id}>
             {attachmentViewObject.actions}
-            <RegisteredAttachmentsPropertyActions
-              isLoading={isLoading}
-              onDelete={() => handleDeleteComment(comment.id)}
-            />
+            {!attachmentViewObject.hideDefaultActions && (
+              <RegisteredAttachmentsPropertyActions
+                isLoading={isLoading}
+                onDelete={() => handleDeleteComment(comment.id)}
```

Comment:
Should we allow deleting a file via the attachment?

Reply:
Also, the `hideDefaultActions` attribute hides the delete action from files.